test/imagekit/imagekit_test.rb

```diff
@@ -226,5 +263,15 @@
   
       expect(url).to eq("https://imagekit.io/your-imgekit-id/tr:h-300,w-400/default-image.jpg?ik-sdk-version=ruby-#{Imagekit::Sdk::VERSION}")
     end
+
+    it "get_authentication_params_test_with_hard_coded_params" do
+      SUT = ImageKit::ImageKitClient.new(PRIVATE_KEY, PUBLIC_KEY, URL_ENDPOINT)
+      # SUT.set_ik_request(request_obj)
+
+      result=SUT.get_authentication_parameters('your_token',1582269249)
+      expect('your_token').to eq(result[:token])
+      expect(nil).not_to eq(result[:expire])
+      expect('2b6b30751172436abba7dd9e71aa9dc9410684ce').to eq(result[:signature])
```

Comment:
As per [SDK specification](https://docs.imagekit.io/api-reference/api-introduction/sdk#want-to-contribute) we should have the following values.
```

/*
  Assert that authenticationParameters is same as below JSON. Hard code it in test cases
  { 
    token: 'your_token',
    expire: 1582269249,
    signature: 'e71bcd6031016b060d349d212e23e85c791decdd'
  }
*/
```